x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -9,36 +9,45 @@ import { TypeOf } from '@kbn/config-schema';
 import { HostIsolationRequestSchema } from '../schema/actions';
 
 export type ISOLATION_ACTIONS = 'isolate' | 'unisolate';
-
 export interface EndpointAction {
-  action_id: string;
-  '@timestamp': string;
-  expiration: string;
-  type: 'INPUT_ACTION';
-  input_type: 'endpoint';
-  agents: string[];
-  user_id: string;
-  data: {
-    command: ISOLATION_ACTIONS;
-    comment?: string;
+  type: 'action';
```

Comment:
I don't think we should do this at the top-level type here. the `item` nesting is specific to the audit log. The `EndpointAction` reflects what gets created, what is actually stored, etc. Don't modify `EndpointAction` for this

Probably create a type nearer to the audit log code that just nests it:

```ts
export interface AuditLogAction {
    type: 'action';
    item: EndpointAction;
}

export interface AuditLogResponse {
    type: 'response';
    item: EndpointActionResponse;
}
```

or perhaps there is a more concise way to do that with unions and a single object. I dunno.

Reply:
done 1f727c32bf56b04f0e9a7ebd4c29a77e8b9c63b7